src/data/reducers.js

```diff
@@ -81,12 +81,9 @@ export function labels(state = {}, action) {
     let newState;
     switch (action.type) {
         case ActionTypes.FETCH_LABELS_IF_NEEDED:
-            if (Object.keys(state.labels.labelList).length === 0) {
-
-            }
             return state;
 
-        case ActionTypes.SET_LABELS:
+        case ActionTypes.SET_LABELS: {
```

Comment:
Add brace to fix “Unexpected lexical declaration in case block” (`no-case-declarations`) lint error.